src/connectedk8s/azext_connectedk8s/commands.py

```diff
@@ -29,6 +29,8 @@ def load_command_table(self, _):
         g.custom_command('disable-features', 'disable_features', is_preview=True)
         g.custom_command('list', 'list_connectedk8s', table_transformer=connectedk8s_list_table_format)
         g.custom_show_command('show', 'get_connectedk8s', table_transformer=connectedk8s_show_table_format)
+        g.custom_command('proxy', 'client_side_proxy_wrapper')
 
     with self.command_group('connectedk8s', connectedk8s_sdk_prev, client_factory=cf_connected_cluster_prev_2021_04_01) as g:
-        g.custom_command('proxy', 'client_side_proxy_wrapper', is_preview=True)
+        pass
```

Comment:
>pass [](http://example.com/codeflow?start=8&length=4)

Do we require this block now ?.

Reply:
we dont require it now, but kept it so that it is easier to add preview commands in future. it wont affect the functionality since we have a pass statement.